### PR TITLE
Wire up company page

### PR DESCRIPTION
**Pull request by MonikaFu**
* Make selectors work with the plots. 
* Implement error handling
* Place two plots next to each other

NOTE: the data seems to be off. It shows no data for Automotive for techmix plot but then the power company names seem to be automotive names. Also, for techmix there exist only data for ownership weights for listed equity. I added two issues in `workflow.pacta.dashboard`: https://github.com/RMI-PACTA/workflow.pacta.dashboard/issues/23 and https://github.com/RMI-PACTA/workflow.pacta.dashboard/issues/22. But also we should look into the test data. Since the issues seem to be connected the data and not the workings of the dashboard, I consider this PR ready for review.

### CODE REVIEW

File: src/json/data_techexposure_company_companies.json [hunk 1]
```diff
@@ -1,23 +1,27 @@
...
+		"port_weight": 0.1156,
+		"allocation": "portfolio_weight",
+		"scenario_source": "WEO2023",
+		"scenario": "APS",
 		"year": 2028,
 		"asset_class": "Listed Equity",
 		"ald_sector_translation": "Automotive",
 		"asset_class_translation": "Listed Equity",
 		"technology_translation": "LDV: Electric"
 	},
```
Comment (jdhoffa) [inline src/json/data_techexposure_company_companies.json:16]: Hyundai appears to be accurately classed as an `Automotive` company in the data here?

**Comment by jdhoffa** [comment]
@MonikaFu I believe the data is correct. In the (test) data, Hyundai is classed as both an Automotive Company and a Power company, when using the `ownership_weight` methodology. This is as expected, as it is likely that Hyundai owns both Car and Power producing assets (and `ownership_weight` allows for dual sectored companies, since we just roll up whatever the production is by owning share). 

In `portfolio_weight`, that wouldn't be the case since we must select a specific sector to tie production to.

E.g.
Screenshot 2024-11-26 at 15 53 37

**Comment by jdhoffa** [comment]
I am curious however why "Hyundai" isn't appearing in the company data plots for the Automotive sector? There is relevant data there, so I am surprised to not see a corresponding plot.

**Comment by MonikaFu** [comment]
thanks @jdhoffa . Then indeed in this case it is not incorrect that automotive companies appear in power sector. It was strange to me to see them in power but not in automotive, that's why I thought it is a problem in the data. It still feels strange that they don't appear in the plots for automotive

**Comment by jdhoffa** [comment]
> thanks @jdhoffa . Then indeed in this case it is not incorrect that automotive companies appear in power sector. It was strange to me to see them in power but not in automotive, that's why I thought it is a problem in the data. It still feels strange that they don't appear in the plots for automotive

Data not appearing in the plots for Automotive feels like a problem that should be tackled in this PR actually! Or at least assessed to understand why they aren't showing up

**Comment by MonikaFu** [comment]
@jdhoffa you are right, I was confused because of the data for Auto showing in Power. I somewhat assumed the data must be wrong. After closer inspection I found a bug in my implementation. I fixed it and added a check to prevent it in the future.

**Review by jdhoffa — APPROVED** [review]

**Pull request merged**